hera_cal/abscal.py

```diff
@@ -315,13 +318,80 @@ def delay_lincal(self, medfilt=True, kernel=(1, 11), time_ax=0, freq_ax=1, verbo
                     fit[phi_key] = np.repeat(phi_avg, Ntimes, axis=time_ax)
 
         # form result
-        self._ant_dly = odict(map(lambda k: (k, copy.copy(fit["tau_{}_{}".format(k[0], k[1])])), self._flatten(self._gain_keys)))
+        self._ant_dly = odict(map(lambda k: (k, copy.copy(fit["tau_{}_{}".format(k[0], k[1])])), flatten(self._gain_keys)))
         self._ant_dly_arr = np.moveaxis(map(lambda pk: map(lambda k: self._ant_dly[k], pk), self._gain_keys), 0, -1)
 
         if solve_offsets:
-            self._ant_dly_phi = odict(map(lambda k: (k, copy.copy(fit["phi_{}_{}".format(k[0],k[1])])), self._flatten(self._gain_keys)))
+            self._ant_dly_phi = odict(map(lambda k: (k, copy.copy(fit["phi_{}_{}".format(k[0],k[1])])), flatten(self._gain_keys)))
             self._ant_dly_phi_arr = np.moveaxis(map(lambda pk: map(lambda k: self._ant_dly_phi[k], pk), self._gain_keys), 0, -1)
 
+    def delay_slope_lincal(self, medfilt=True, kernel=(1, 11), time_ax=0, freq_ax=1, verbose=True, time_avg=False,
+                            four_pol=False):
+        """
+        Solve for per-antenna delay according to the equation
```

Comment:
Update documentation. It's really unclear how this is different from the other delay lincal.